src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -128,7 +129,13 @@
         } else {
             if (password!=null) {
                 String authorization = (login + ':' + password);
-                encodedAuthorization = "Basic "+new String(Base64.encodeBase64(authorization.getBytes()));
+                final Charset charset;
+                try {
+                    charset = Charset.forName("UTF-8");
```

Comment:
GitHub API is being widely used outside Jenkins. It definitely makes sense to consider the migration, but it needs a wider discussion or a decision from the library owner (@kohsuke)